docs/cloud-storage-fileset-example.md

```diff
@@ -0,0 +1,676 @@
+bin/gravitino.sh start
+```
+
+### Bundle jars
+
+`gravitino-{aws,gcp,aliyun,azure}-hadoop-bundle` are the jars that contain all the necessary classes to access the corresponding cloud storages, for instance, `gravitino-aws-hadoop-bundle.jar` contains the all necessary classes including `hadoop-common`(hadoop-3.3.1) and `hadoop-aws` to access the S3 storage.
+**They are used in the scenario where there is no hadoop environment in the runtime.**
+
+**If there is already hadoop environment in the runtime, you can use the `gravitino-{aws,gcp,aliyun,azure}-bundle.jar` that does not contain the cloud storage classes (like hadoop-aws) and hadoop environment, you can manually add the necessary jars to the classpath.**
+
+The following table demonstrates what jars are necessary for different cloud storage filesets:
+
+| Hadoop runtime version | S3                                                                                                                                     | GCS                                                                                                            | OSS                                                                                                                                             | ABS                                                                                                                 |
+|------------------------|----------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| No Hadoop environment  | gravitino-aws-hadoop-bundle-{gravitino-version}.jar                                                                                    | gravitino-gcp-hadoop-bundle-{gravitino-version}.jar                                                            | gravitino-aliyun-hadoop-bundle-{gravitino-version}.jar                                                                                          | gravitino-azure-hadoop-bundle-{gravitino-version}.jar                                                               |
+| 2.x, 3.x               | gravitino-aws-bundle-{gravitino-version}.jar, hadoop-aws-{hadoop-version}.jar, aws-sdk-java-{version} and other necessary dependencies | gravitino-gcp-bundle-{gravitino-version}.jar, gcs-connector-{hadoop-version}.jar, other necessary dependencies | gravitino-aliyun-bundle-{gravitino-version}.jar, hadoop-aliyun-{hadoop-version}.jar, aliyun-sdk-java-{version} and other necessary dependencies | gravitino-azure-bundle-{gravitino-version}.jar, hadoop-azure-{hadoop-version}.jar, and other necessary dependencies |
```

Comment:
Why do you use a table, it is hard to read.

Reply:
I looks acceptable in the markdown format. 

<img width="1501" alt="image" src="https://github.com/user-attachments/assets/139de7d5-0c3b-4db2-9ac4-b7de8af71356" />

Do you have any good suggestions to show the content?